### PR TITLE
feat: 360 video icon

### PR DESCRIPTION
**Pull request by aronshamash**
- https://nidigitalsolutions.jira.com/browse/TOC-29
- New 360 video play button designs https://app.zeplin.io/project/5c77fbe0a481f9bcdf61adb5/dashboard

### CODE REVIEW

File: packages/video/src/inline-video-player.web.js [hunk 1]
```diff
@@ -3,23 +3,54 @@ import React, { Component } from "react";
 import { propTypes, defaultProps } from "./video-prop-types";
 import SkySportsBanner from "./sky-sports-banner";
 
+const static360PlayIcon =
```
Comment (jimhunty) [inline packages/video/src/inline-video-player.web.js]: Should be in the Icons package.

File: packages/video/src/inline-video-player.web.js [hunk 2]
```diff
@@ -3,23 +3,54 @@ import React, { Component } from "react";
...
   border-style: solid;
   border-width: 3px;
   border-color: white;
 }
 
+.video-js video:not([data-player="default"]) ~ button span {
+  display: none;
+}
+
+.video-js video:not([data-player="default"]) ~ button, .video-js video:not([data-player="default"]) ~ button:hover {
```
Comment (jimhunty) [inline packages/video/src/inline-video-player.web.js]: 🤯

Comment (nathggns) [reply packages/video/src/inline-video-player.web.js]: Little bit hacky, but it's the only way we can think of to know whether the video is 360 or not.

**Comment by nehasri89** [comment]
can you put a screenshot for this pls @aronshamash

**Pull request closed without merging**